src/test/java/blackjack/domain/result/PlayerProfitsTest.java

```diff
@@ -0,0 +1,57 @@
+package blackjack.domain.result;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import blackjack.domain.bet.Profit;
+import blackjack.domain.player.Player;
+import blackjack.domain.player.TestPlayerCreator;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("플레이어의 수익 도메인 테스트")
+class PlayerProfitsTest {
+
+    @DisplayName("특정 플레이어의 수익을 조회할 수 있다")
+    @Test
+    void testFindProfitOfPlayer() {
+        Player player1 = TestPlayerCreator.of("썬", 1, 2, 3, 4);
+        Player player2 = TestPlayerCreator.of("리비", 3, 4);
+
+        Map<Player, Profit> playerProfitMap = new HashMap<>();
```

Comment:
기본적으로 변수명에는 자료구조를 사용하지 않습니다~
클린코드의 의미 있는 이름에 대해 학습해보아요
```suggestion
        Map<Player, Profit> playerProfits = new HashMap<>();
```

Reply:
특정 자료형에 종속적인 변수명은 의미를 드러내지 못하고 확장사항에 잘 견디지 못하는 것을 학습할 수 있었습니다 
반영해보겠습니다 💪🏻